webapp/lib/view.dart

```diff
@@ -113,6 +115,130 @@ class DashboardView {
   }
 }
 
+enum FormGroupTypes {
+  TEXT,
+  CHECKBOX,
+  DATE
+}
+
+class ProjectConfigurationView {
+  DivElement configurationViewElement;
+  FormElement _projectConfigurationForm;
+
+  ProjectConfigurationView() {
+    configurationViewElement = new DivElement()
+      ..classes.add('project-configuration');
+    _projectConfigurationForm = new FormElement()
+      ..classes.add('configuration-form');
+    _buildForm();
+    configurationViewElement.append(_projectConfigurationForm);
+  }
+
+  void _buildForm() {
+    _projectConfigurationForm
+      ..append(
+        _multipleFormGroup('Project Languages:',
+          {'English': {'send': FormGroupTypes.CHECKBOX, 'receive': FormGroupTypes.CHECKBOX },
+            'Somali': {'send': FormGroupTypes.CHECKBOX, 'receive': FormGroupTypes.CHECKBOX }})
```

Comment:
We won't know in advance what languages the project will be in, so the languages themselves need to be configurable, not just whether they're send/receive. I suggest we use the same "+" button" as we do for adding new sms-es to add a new language, with a textbox for inputting the language (for now)

Reply:
As discussed earlier, I'll do this in a separate PR.